app/models/project.rb

```diff
@@ -1,4 +1,6 @@
 class Project < ApplicationRecord
+  TENKFT_ID_FOR_SUPPORT = 2360793
```

Comment:
```suggestion
  TENK_ID_FOR_SUPPORT = 2360793
```

For consistency with other uses of `tenk` to mean 10kft.